frontend/src/public/css/style.css

```diff
@@ -19,4 +19,25 @@
 .card-row{
     color: #283583; 
     margin: auto
+}
+
+body {
+    display: flex;
+    min-height: 100vh;
+    flex-direction: column;
+}
+
+main {
+flex: 1 0 auto;
+}
+
+.list-inline {
+    padding-left: 0;
+    margin-left: -5px;
```

Comment:
May want to be cautious of using `px` units; on large displays it will end up being unnoticeable and on small displays it can be invasive.

Reply:
Yeah I'll keep that in mind. I tested the page on different devices (mobile, tablets etc) and it seems to be responsive.